### PR TITLE
Video-27-Create-Place-Order-API

### PR DESCRIPTION
**Pull request by basir**

### CODE REVIEW

**Comment by ghost** [comment]
Please shear the code sir.

File: backend/utils.js [hunk 1]
```diff
@@ -14,3 +14,24 @@ export const generateToken = (user) => {
     }
   );
 };
+
+export const isAuth = (req, res, next) => {
+  const authorization = req.headers.authorization;
+  if (authorization) {
+    const token = authorization.slice(7, authorization.length); // Bearer XXXXXX
```
Comment (dream903) [inline backend/utils.js:21]: xhr.js:210          POST http://localhost:3001/api/orders 401 (Unauthorized)
message :INVALID TOKEN
export const isAuth=(req,res,next)=>{
    const authorization=req.headers.authorization;


    if(authorization){
        const token=authorization.slice(7,authorization.length);
        jwt.verify(token,process.env.JWT_SECRET||'somethingsecret',
        (err,decode)=>{
            if(err){
                res.status(401).send({message:'invalid token'});
            }else{
                req.user=decode;
                next();
            }
        }
        );
    }
    else{
        res.status('401').send({message:'No Token'});
    }
};

Comment (King23WL) [reply backend/utils.js:21]: Hello! Did you fix this one?

Comment (layiwola008) [reply backend/utils.js:21]: Check your createOrder in orderActions.js and make you spell the headers as headers (with an 's'). Make sure there is no typo in the file.

**Pull request merged**